autorest/client_test.go

```diff
@@ -595,3 +605,13 @@ func TestResponseGetPollingLocation(t *testing.T) {
 			mocks.TestURL, ar.GetPollingLocation())
 	}
 }
+
+func randomString(n int) string {
+	const chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
don't seed the global rand never ever. you're making other tests running in the same process which might be relying on the fixed seed of the global rand. if you want to construct a rand instance:

    r := rand.New(rand.NewSource(time.Now().UTC().UnixNano()))